webapi/src/main/scala/org/knora/webapi/messages/util/search/gravsearch/prequery/GravsearchQueryOptimisation.scala

```diff
@@ -141,18 +102,18 @@ class RemoveRedundantKnoraApiResourceOptimisationFeature(
  * use with a property that can only be used with that type (unless the property
  * statement is in an `OPTIONAL` block).
  */
-class RemoveEntitiesInferredFromPropertyOptimisationFeature(
-  typeInspectionResult: GravsearchTypeInspectionResult,
-  querySchema: ApiV2Schema
-) extends GravsearchQueryOptimisationFeature(typeInspectionResult, querySchema) {
+private object RemoveEntitiesInferredFromProperty {
 
   /**
    * Performs the optimisation.
    *
    * @param patterns the query patterns.
    * @return the optimised query patterns.
    */
-  override def optimiseQueryPatterns(patterns: Seq[QueryPattern]): Seq[QueryPattern] = {
+  def removeEntitiesInferredFromProperty(
+    patterns: Seq[QueryPattern],
+    typeInspectionResult: GravsearchTypeInspectionResult
```

Comment:
Either remove params from Scaladocs or update them.

Reply:
Will do in a future PR: 9f04f05c6320d12c84a840fe5d99a49f29f99703